sessions/redis.go

```diff
@@ -22,14 +22,22 @@ type RedisStore interface {
 //
 // It is recommended to use an authentication key with 32 or 64 bytes. The encryption key,
 // if set, must be either 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256 modes.
-func NewRedisStore(size int, network, address, password string, keyPairs ...[]byte) (RedisStore, error) {
+func NewRediStore(size int, network, address, password string, DB string, keyPairs ...[]byte) (RedisStore, error) {
 	store, err := redistore.NewRediStore(size, network, address, password, keyPairs...)
 	if err != nil {
 		return nil, err
 	}
 	return &redisStore{store}, nil
 }
 
+func NewRediStoreWithDB(size int, network, address, password string, DB string, keyPairs ...[]byte) (RedisStore, error) {
```

Comment:
this however can be a good change / proposal, but for keeping the naming, I would prefer it would be changed to `NewRedisStoreWithDB`